fiftyone/core/collections.py

```diff
@@ -10476,8 +10548,9 @@ def aggregate(self, aggregations, _mongo=False):
                 instances
 
         Returns:
-            an aggregation result or list of aggregation results corresponding
-            to the input aggregation(s)
+            The aggregation result(s) corresponding to the input aggregation(s).
+            Returns a single result for a single aggregation, a list of results
+            for multiple aggregations, or a generator if all aggregations are lazy
```

Comment:
nit: [style guide](https://github.com/voxel51/fiftyone/blob/develop/STYLE_GUIDE.md#python-style-guide) stays line limit is 79 characters. This last line is too long

I will never give up on pointing this out until/unless the rule is formally changed.

Reply:
We should add this to the GitHub Actions. This type of style check can be easily automated.

Reply:
Ideally to pre-commit hooks, yes. That would make me so happy!

Unfortunately our `black` linter doesn't do comments, only code. I presume there are some subtleties that make auto-formatting comments difficult.

Reply:
Yeah `pylint` can do the line-length where `black` falls down

Reply:
That would be great, please add that!